### DISCUSSION TITLE
Cannot start webapp

### DISCUSSION THREAD

**Discussion by Kastlet** [searx/searx#3088] [Installation issue: Official wiki] [answered]
I'm trying to install searx using the step by step installation, but I cannot start webapp due to this error
![image](https://user-images.githubusercontent.com/78056798/145260603-bf515921-59f5-4ba6-b975-fe8066b292d6.png)

**Comment by virtadpt** [comment]
What are the permissions and ownership on that file?

`ls -alF /usr/local/searx/searx-sc/searx/settings.yml`

**Comment by Kastlet** [comment]
![image](https://user-images.githubusercontent.com/78056798/145726065-e7a81520-5ecf-4cf5-8375-497b0c13e663.png)
this shows up after I run the command, there isn't anything weird with the permissions (also I'm starting to wonder if this is because im using WSL)

**Comment by Kastlet** [comment, marked as answer]
I have fixed this by reinstalling the Ubuntu WSL app
